OpenRA.Mods.Common/Activities/Air/HeliFly.cs

```diff
@@ -61,9 +62,6 @@ public override Activity Tick(Actor self)
 				return NextActivity;
 			}
 
-			if (IsCanceled || !target.IsValidFor(self))
```

Comment:
Why was this removed?

Reply:
I was confused while rebase-merging: target field became Target. I'll put this one back in.

Reply:
It is back in.